src/model/entity/level.go

```diff
@@ -1,8 +1,98 @@
 package entity
 
+import (
+	"errors"
+	"math/rand"
+)
+
+const (
+	RoomMinWidth     = 3
+	RoomMinHeight    = 3
+	MinRoomPadding   = 1
+	numberXYSections = 3
+)
+
 type Level struct {
-	Rooms       []Room
-	Passages    []Passage
-	LevelNumber uint
-	LevelEnd    Box
+	Rooms    []Room
+	Passages []Passage
+	Number   uint
+	End      Box
+}
+
+func calculateRoomSectionSize(sizeMap Size2D[uint]) (sectionSize Size2D[uint]) {
+	var totalPaddingWidth uint = MinRoomPadding * 2
+	var totalPaddingHeight uint = MinRoomPadding * 2
```

Comment:
Почему здесь нужно сделать через длинную запись инициализации, а не просто?

```suggestion
	totalPaddingWidth := MinRoomPadding * 2
	totalPaddingHeight := MinRoomPadding * 2
```

Я ничего против не имею, просто интересно чем здесь вызвано такое решение. Когда ниже идёт такая же инициализация, но уже через коротную запись.

Reply:
Я так сделала, потому что в Go нет неявного приведения и я решила не в вычислении ниже привести к uint, а заранее при объявлении
<img width="628" height="152" alt="image" src="https://github.com/user-attachments/assets/18542c0d-edad-4cb4-ab18-9d5fe2597ff6" />
Но не подумала, что это разные стили. Наверное лучше сделать так

> totalPaddingWidth := uint(MinRoomPadding * 2)
	totalPaddingHeight := uint(MinRoomPadding * 2)

P.s.
А вот с глобальными const переменным Go как-то справляется без красных предупреждений -.- (uint) поделил на (int)
<img width="634" height="262" alt="image" src="https://github.com/user-attachments/assets/46ab63f5-fc95-4eac-accf-0041bc77501a" />

Reply:
Ну да, интересно. Думаю, лучше сделать через явное приведение к типу, а не через объявление

Reply:
Не очень поняла, как в итоге делать 😅
Так:

> totalPaddingWidth := uint(MinRoomPadding * 2)
totalPaddingHeight := uint(MinRoomPadding * 2)

Так:
> var totalPaddingWidth uint = MinRoomPadding * 2
var totalPaddingHeight uint = MinRoomPadding * 2

Так:
> totalPaddingWidth := MinRoomPadding * 2
	totalPaddingHeight := =MinRoomPadding * 2
	availableWidth := mapSize.Width - uint(totalPaddingWidth)
	availableHeight := mapSize.Height - uint(totalPaddingHeight)

Reply:
Первый вариант:

```
totalPaddingWidth := uint(MinRoomPadding * 2)
totalPaddingHeight := uint(MinRoomPadding * 2)
```